<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QGridLayout" name="main_layout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="_cards_table_view_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="input_table_placeholder">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">padding-top: 10px;
padding-bottom: 3px;</string>
        </property>
        <property name="text">
         <string>Cards already in deck (showing 13/24):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="deck_preview_table_placeholder"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="_deck_info_layout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLineEdit" name="deck_info_title_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: white white black white ;</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>New Deck</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deck_info_save_button">
        <property name="text">
         <string/>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deck_info_path_label">
        <property name="styleSheet">
         <string notr="true">color: #666</string>
        </property>
        <property name="text">
         <string>path/to/the/deck.json</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>34</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="app_status_bar_placeholder"/>
  <widget class="QMenuBar" name="_top_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="_top_menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="top_menu_file_new"/>
    <addaction name="top_menu_file_open"/>
    <addaction name="separator"/>
    <addaction name="top_menu_file_save"/>
    <addaction name="top_menu_file_save_as"/>
    <addaction name="top_menu_file_export"/>
   </widget>
   <widget class="QMenu" name="_top_menu_tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="top_menu_tools_settings"/>
   </widget>
   <addaction name="_top_menu_file"/>
   <addaction name="_top_menu_tools"/>
  </widget>
  <action name="top_menu_file_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="top_menu_file_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="top_menu_file_open">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="top_menu_file_export">
   <property name="text">
    <string>Export to Anki...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="top_menu_file_new">
   <property name="text">
    <string>New deck...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menu_toggle_sidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Tools sidebar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="top_menu_tools_show_shortcuts">
   <property name="text">
    <string>Show shortcuts</string>
   </property>
  </action>
  <action name="top_menu_tools_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
